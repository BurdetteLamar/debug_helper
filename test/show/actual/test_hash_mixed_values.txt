---
Hash (size=3 message='test_hash_mixed_values'):
  size: 3
  Pair 0:
    Key:
      Symbol:
        to_s: a
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value: Fixnum 0
  Pair 1:
    Key:
      Symbol:
        to_s: b
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value:
      String:
        to_s: '0'
        size: 1
        encoding: !ruby/encoding UTF-8
        ascii_only?: true
        bytesize: 1
  Pair 2:
    Key:
      Symbol:
        to_s: c
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value: NilClass nil
