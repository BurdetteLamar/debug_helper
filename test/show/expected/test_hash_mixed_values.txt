---
Hash (message='test_hash_mixed_values'):
  Hash#size: 3
  Hash#default: 
  Hash#default_proc: 
  Pair 0:
    Key:
      Symbol:
        Symbol#to_s: a
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Integer 0
  Pair 1:
    Key:
      Symbol:
        Symbol#to_s: b
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value:
      String:
        String#to_s: '0'
        String#size: 1
        String#encoding: !ruby/encoding UTF-8
        String#ascii_only?: true
        String#bytesize: 1
  Pair 2:
    Key:
      Symbol:
        Symbol#to_s: c
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: NilClass nil
